include/openvizsla.h

```diff
@@ -25,9 +25,10 @@ struct
 #endif
 ov_packet {
 	uint8_t  magic;
-	uint16_t flags;
+	uint8_t flags;
 	uint16_t size;
```

Comment:
Can we also have 
```C
uint8_t timestamp_len : 3;
uint16_t size : 13;
```
?

Reply:
Why would you need `timestamp_len` later? Note that bitfields are not really portable so you cannot really take any advantage with bitfields. after the cumulative timestamp is calculated it is always 64-bit and the encoded delta timestamp is of no use.

Reply:
Agree